src/fragments/lib/auth/js/social.mdx

```diff
@@ -4,6 +4,10 @@
 
 OAuth support in Amplify uses Cognito User Pools and supports federation with social providers, which will automatically create a corresponding user in the User Pool after a login. [OIDC](https://en.wikipedia.org/wiki/OpenID_Connect) tokens are available in the app after the application has completed this process.
 
+<Callout warning>
+ When configuring a Social sign-in through the Amplify CLI, it's important to exercise caution when designating attributes as "required." Different social identity providers have varied scopes in terms of the information they respond back to Cognito with.  User pool attributes that are initial set up as "required" cannot be changed later, and may require you to migrate the users or create a new user pool.  
```

Comment:
grammar nit:
```suggestion
 When configuring Social sign-in through the Amplify CLI, it's important to exercise caution when designating attributes as "required." Different social identity providers have varied scopes in terms of the information they respond back to Cognito with.  User pool attributes that are initial set up as "required" cannot be changed later, and may require you to migrate the users or create a new user pool.  
```

Reply:
Nice catch Scott! Updated it in most recent commit where I added the callout across all platforms. Also lower-cased that "Social" at beginning.